spec/spec_helper.rb

```diff
@@ -1,10 +1,10 @@
 require 'rubygems'
-require 'spork'
+#require 'spork'
 $: << File.dirname(File.dirname(__FILE__))
 #uncomment the following line to use spork with the debugger
 #require 'spork/ext/ruby-debug'
 
-Spork.prefork do
+#Spork.prefork do
```

Comment:
Incorrect indentation detected (column 0 instead of 2).<br>Missing space after #.